app/views/users/index.html.erb

```diff
@@ -8,6 +8,7 @@
 	     <p><%= user.name %></p>
 			 <p> <%= image_tag user_avatar(user) , class: "img-fluid med-profile-img" %> </p>
 			 <%= link_to "See Profile", user, :class => "bg-red font-white p-1 profile" %>
+			 <%= button_to "Remove Friend",{:controller => 'friendships', :action => 'destroy' :id => friendship.id }, {:method => :patch, :class => "bg-red font-white p-1 acceptingfriend"} %>
```

Comment:
We don't have 'Friendship:destroy' action, the friendship is being confirmed after clicking the `reject friend` button. Please implement the `Friendship:destroy` action and use delete HTTP verb as opposed to patch.

Let's put a comma after `:action => 'destroy'` to fix a syntax error preventing the apllication from rendering Users page.